config/application.yml.default

```diff
@@ -107,6 +107,7 @@ encrypted_document_storage_s3_bucket: ''
 event_disavowal_expiration_hours: 240
 feature_idv_force_gpo_verification_enabled: false
 feature_idv_hybrid_flow_enabled: true
+feature_new_device_alert_aggregation: true
```

Comment:
Minor: We have a few different naming conventions for enabling features, and this seems to be an outlier.

Existing:

- `enable_[feature_name]`
- `feature_[feature_name]_enabled`

This seems close to the second pattern, except it doesn't suffix with `_enabled`